composer.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "placidapp/placid-php",
-    "description": "PHP Wrapper around the Placid.app REST API",
+    "name": "jurry/placid-php",
```

Comment:
Is this intended to change the `name` and `description`?

Reply:
Seems no, let me change it

Reply:
Changes on `composer.json` have been reverted.